ProjectLighthouse.Servers.Website/Pages/UserSettingsPage.cshtml.cs

```diff
@@ -33,6 +33,8 @@ public async Task<IActionResult> OnPost([FromRoute] int userId, [FromForm] strin
 
         if (avatarHash != null) this.ProfileUser.IconHash = avatarHash;
 
+        if (this.User.IsAdmin) this.ProfileUser.ProfileTag = profileTag;
```

Comment:
Does this also need to check if profile tag isn't null?

Reply:
No - if it's null or whitespace it just isn't rendered